pycbc/filter/matchedfilter.py

```diff
@@ -257,13 +257,14 @@ def full_matched_filter_and_cluster_symm(self, segnum, template_norm, window):
         self.ifft.execute()
         snrv, idx = self.threshold_and_clusterers[segnum].threshold_and_cluster(self.snr_threshold / norm, window)
 
-        if len(idx) == 0:
```

Comment:
I think we want to keep these actually (I'm not sure of all the donwstream problems of changing this atm), but should amend the documentation to say that only empty lists are returned in the case of no above threshold triggers. For testing, one can always lower the threshold arbitrarily.